sklearn/utils/estimator_checks.py

```diff
@@ -3925,6 +3960,27 @@ def check_n_features_in_after_fitting(name, estimator_orig):
     ]
     X_bad = X[:, [1]]
 
+    err_msg = (
+        "`{name}.{method}()` does not check for consistency between input number "
```

Comment:
with these long comments, I want to make it very easy for people to fix issues and as as a result follow our API. I'll create more PRs extending our error messages with fix suggestions.

Reply:
Instead of using `"\n"` everywhere, can we use this pattern:

```python
    err_msg = textwrap.dedent("""\
    This is
    my
    multiple
    line error message
    """)
```

Reply:
I've used this format in other places I think, but I find it very ugly, since due to indentation it needs to be sth like:

```py
def g():
    err_msg = """This
is
my
message"""
    print(err_msg)
```

Reply:
With `textwrap.dedent` you can indent without any issues.

Reply:
Done, but note that this adds a newline at the beginning and the end of the message (not important to me)